python/requirements.txt

```diff
@@ -1,4 +1,5 @@
 six
+boto==2.38.0
```

Comment:
Errrr? You sure about that? 

We could also have a `requirements-dev.txt`, if we want to expand the scope of requirements.